### PR TITLE
Adding CCHeaderService using ATHeaderService as template.

### PR DESCRIPTION
**Pull request by menanteau**

### CODE REVIEW

File: sal_interfaces/CCHeaderService/CCHeaderService_Events.xml [hunk 1]
```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="http://lsst-sal.tuc.noao.edu/schema/SALEventSet.xsl"?>
+<SALEventSet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALEventSet.xsd">
+<SALEvent>
+    <Subsystem>CCHeaderService</Subsystem>
+    <Version>3.8</Version>
+    <Author></Author>
+    <EFDB_Topic>CCHeaderService_logevent_offlineDetailedState</EFDB_Topic>
```
[the remark this answers is not in the document]
Comment (menanteau) [reply sal_interfaces/CCHeaderService/CCHeaderService_Events.xml]: Should I update `ATHeaderService_Events.xml` with these changes too?

[the remark this answers is not in the document]
Comment (mareuter) [reply sal_interfaces/CCHeaderService/CCHeaderService_Events.xml]: If you want to clean that up too, go ahead.

[the remark this answers is not in the document]
Comment (menanteau) [reply sal_interfaces/CCHeaderService/CCHeaderService_Events.xml]: Made the changes you requested and removed `offlineDetailedState` from both `AT` and `CC`

**Comment by rbovill** [comment]
In order for the tests to pass, one more change must be made. Please add CCHeaderService to the subsystems list in python/lsst/ts/xml/testutils.py.  This will allow the unit tests to run on the new XML files and point to any other needed changes.

**Comment by menanteau** [comment]
Adding CCHeaderService using ATHeaderService as template

**Pull request merged**